lab.R

```diff
@@ -21,50 +21,51 @@
 library(tidyverse)
 library(skimr)
 library(visdat)
-
+library(testthat)
 library(AmesHousing)
 
 dataf = read_csv(file.path('data', 'ames.csv'))
 
 ## To check your answers locally, run the following: 
-## testthat::test_dir('tests')
+testthat::test_dir('tests')
```

Comment:
This is causing an infinite loop:  The first thing the test file does is run your script, and this line runs the test file.  So leave this commented out in your script.  I've definitely realized that, next time around, I need to explain this infrastructure more carefully for the first lab. 
```suggestion
# testthat::test_dir('tests')
```

Reply:
Oh  see! Thank you Dan, that solved it